examples/dem/3d-bouncing-particle/Bisection.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def Young_effectif(Poisson_wall, Young_wall, Poisson_particule, Young_particule):
+    # Calculate the effective Young modulus between two material
+    Y_effectif = ( (1-Poisson_wall**2)/Young_wall  + (1-Poisson_particule**2)/Young_particule )**(-1)
+    return Y_effectif
+
+
+def bisection(Poisson_wall, Young_wall, Poisson_particule , tol, N_max,mass_particule, R_particule, Y_estim1, Y_estim2,kn_voulu):
```

Comment:
Was it necessary to recode a bissection? Couldnt you have just used https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.bisect.html